packages/notionable/src/global/logic.ts

```diff
@@ -45,13 +46,20 @@ export async function onNetworkPending(selectedNetwork: Network) {
 
 export async function onAccountPending(
   selectedAccount: string,
-  selectedNetwork: Network
+  selectedNetwork: Network,
+  walletLabel: string
 ) {
   const isAccountReady = await new Promise<boolean>((resolve) => {
     Registry.getAccountRegistry().onAccountReady(
       selectedNetwork,
       selectedAccount,
-      () => {
+      (a) => {
+        const tokenBalances = a.balances.filter((b) => b.tokenType === 'Underlying').map((data) => {
+          return {
+            [data.token.symbol]: data.toFloat(),
+          }
+        })
+        identify(a.address, selectedNetwork, walletLabel, JSON.stringify(tokenBalances));
```

Comment:
Did you delete the other `identify` call? I think it is being called in this file somewhere else.

Reply:
I didn't see another identify call in the file